src/core/font_substitutions.js

```diff
@@ -323,6 +327,48 @@ function getStyleToAppend(style) {
   return "";
 }
 
+function getFamilyName(str) {
+  // See https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/main/src/fcname.c#L137.
```

Comment:
To avoid the link becoming "dead" with any future code changes, wouldn't this be better?
```suggestion
  // See https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/14d466b30a8ab4a9d789977ed94f2c30e7209267/src/fcname.c#L137-170
```